examples/java-matter-controller/java/src/com/matter/controller/commands/pairing/PairOnNetworkLongImInvokeCommand.java

```diff
@@ -0,0 +1,90 @@
+package com.matter.controller.commands.pairing;
+
+import chip.devicecontroller.ChipDeviceController;
+import chip.devicecontroller.GetConnectedDeviceCallbackJni.GetConnectedDeviceCallback;
+import chip.devicecontroller.InvokeCallback;
+import chip.devicecontroller.model.InvokeElement;
+import com.matter.controller.commands.common.CredentialsIssuer;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public final class PairOnNetworkLongImInvokeCommand extends PairingCommand {
+  private static final int MATTER_PORT = 5540;
+  private long devicePointer;
```

Comment:
Can the devicePointer field be moved below the other statics just for organization?